tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,899 @@
+      rx pathid: 0, tx pathid: 0x0
+  Refresh Epoch 1
+  65002, (received-only)
+    169.254.0.1 from 169.254.0.1 (169.254.0.1)
+      Origin incomplete, metric 100, localpref 100, valid, external
+      rx pathid: 0, tx pathid: 0
+  Refresh Epoch 1
+  65002
+    169.254.0.57 from 169.254.0.57 (169.254.0.1)
+      Origin incomplete, metric 100, localpref 100, valid, external
+      rx pathid: 0, tx pathid: 0
+```
+
+```
+cisco-asr#sh ip cef 172.16.100.0
+172.16.100.0/24
+  nexthop 169.254.0.1 Tunnel1
+```
+
+### Getting Higher Throughput 
+
+As documented in the [GCP Advanced Configurations](https://cloud.google.com/compute/docs/vpn/advanced), each Cloud VPN tunnel can support up to 3 Gbps when the traffic is traversing a [direct peering](https://cloud.google.com/interconnect/direct-peering) link, or 1.5 Gbps when traversing the public Internet. To increase the VPN throughput the recommendation is to add multiple Cloud VPN gateway on the same region to load balance the traffic across the tunnels. The 2 VPN tunnels configuration example here is built based on the IPsec tunnel and BGP configuration illustrated above, can be expanded to more tunnels if required.
+
+
+#### Cisco ASR Configuration
+
+The ASR 1000 router run cef load balancing based on source and destination ip address hash, each VPN tunnels will be treated as an equal cost path by routing, it can support up to 16 equal cost paths load balancing.
+
+<pre>
+crypto ikev2 keyring <b>VPN_SCALE_TEST_KEY</b>
+
+ peer <b>GCP1</b>
+
+  address <b>104.196.200.68</b>
+
+  pre-shared-key <b>MySharedSecret</b>
+
+ peer <b>GCP3</b>
+
+  address <b>35.185.3.177</b>
+
+  pre-shared-key <b>MySharedSecret</b> 
+
+!
+
+interface <b>Tunnel1</b>
+
+ description VPN tunnel1 to same region GCP for load balancing
+
+ ip address <b>169.254.0.2 255.255.255.252</b>
+
+ ip mtu 1400
+
+ ip tcp adjust-mss 1360
+
+ tunnel source <b>TenGigabitEthernet0/0/0</b>
+
+ tunnel mode ipsec ipv4
+
+ tunnel destination <b>104.196.200.68</b>
+
+ tunnel protection ipsec profile VPN_SCALE_TEST_VTI
+
+!
+
+interface <b>Tunnel3</b>
+
+description VPN tunnel3 to the same region GCP for load balancing
+
+ ip address <b>169.254.0.10 255.255.255.252</b>
+
+ ip mtu 1400
+
+ ip tcp adjust-mss 1360
+
+ tunnel source <b>TenGigabitEthernet0/0/0</b>
+
+ tunnel mode ipsec ipv4
+
+ tunnel destination <b>35.185.3.177</b>
+
+ tunnel protection ipsec profile <b>VPN_SCALE_TEST_VTI_3</b>
+
+!
+
+router bgp <b>65001</b>
+
+ bgp log-neighbor-changes
+
+ neighbor GCP peer-group
+
+ neighbor GCP remote-as <b>65002</b>
+
+ neighbor GCP timers <b>20 60 60</b>
+
+ neighbor <b>169.254.0.1</b> peer-group GCP
+
+ neighbor <b>169.254.0.9</b> peer-group GCP
+
+ !
+
+ address-family ipv4
+
+  network <b>10.0.0.0</b>
+
+  neighbor <b>169.254.0.1</b> activate
+
+  neighbor <b>169.254.0.9</b> activate
+
+  maximum-paths 16
+
+ exit-address-family
+
+!
+</pre>
+
+#### Google Cloud Platform (GCP) Configuration
```

Comment:
You've already established "(GCP)" so don't do it again.

Reply:
And never in headings, anyway.

Reply:
Done